app/views/shared/_user_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for @user do |f| %>
```

Comment:
putting this in `shared` is fine... but it relates fairly specifically to the user model and views, so I would tend to keep this in `/views/users`